src/components/post/api.ts

```diff
@@ -70,23 +70,77 @@ const DUMMY_DATA = {
           musicName: 'Ditto',
           thumbnailUrl: 'url',
           singer: '뉴진스',
-          genre: 'k-pop',
+          genre: '인디 / 어쿠스틱',
+        },
+        likeCount: 10,
+        isBattlePossible: false,
+        nickname: 'Hype',
+      },
+      {
+        postId: 7,
+        music: {
+          musicName: 'Ditto',
+          thumbnailUrl: 'url',
+          singer: '뉴진스',
+          genre: '발라드',
         },
         likeCount: 10,
         isBattlePossible: true,
         nickname: 'Hype',
       },
+      {
+        postId: 8,
+        music: {
+          musicName: 'Ditto',
+          thumbnailUrl: 'url',
+          singer: '뉴진스',
+          genre: '발라드',
+        },
+        likeCount: 10,
+        isBattlePossible: false,
+        nickname: 'Hype',
+      },
     ],
   },
 };
 
-export const getPostFeedData = async () => {
+export const getPostFeedData = async (genre: string, possible: boolean) => {
   try {
     // const res = await axios.get('url', {
     //   headers: { 'Content-Type': 'application/json' },
     // });
 
-    return DUMMY_DATA.data.posts;
+    if (genre === 'all') {
+      if (possible === true) return DUMMY_DATA.data.posts.filter((_, index) => index % 2 === 0);
+      else if (possible === false) return DUMMY_DATA.data.posts.filter((_, index) => index % 2 === 1);
```

Comment:
```suggestion
      possible ? DUMMY_DATA.data.posts.filter((_, index) => index % 2 === 0) : DUMMY_DATA.data.posts.filter((_, index) => index % 2 === 1);
```
삼항 연산자를 사용해서 조금 간단히 작성하는 방법도 있을 것 같아요!